# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/xpconnect/tests/mochitest/bug1681664_helper.js
# Commit: 540c9159c3c6
# Full Hash: 540c9159c3c6dd350ac5343f9f9c977e4cdc53df
# Author: yulia <ystartsev@mozilla.com>
# Date: 2020-12-18 21:38:02
# Regressor Bug: 1681664
# File Overlap Count: 2
# Description:
#   Bug 1681664 - Allow Top-level await modules which fail to evaluate due to user action to fail without crashing. r=jonco,mconley
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D99809
# ==============================================================================

diff -r 02e1e9b5931a -r 540c9159c3c6 js/xpconnect/tests/mochitest/bug1681664_helper.js
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/js/xpconnect/tests/mochitest/bug1681664_helper.js	Fri Dec 18 12:37:36 2020 +0000
@@ -0,0 +1,1 @@
+while (true) {};